drivers/sensor/ags10/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Balthazar Deliers
+# SPDX-License-Identifier: Apache-2.0
+
+# AOSONG AGS10 TVOC sensor driver options.
+
+config AGS10
+	bool "AOSONG AGS10 TVOC sensor"
+	default y
+	select I2C

```

Comment:
Add a `depends on DT_HAS_AOSONG_AGS10_ENABLED` --along with `default y` this will cause the driver to get enabled automatically if the sensor appears in the devicetree (and left out otherwise)

Reply:
done